packages/editor-ui/src/components/ExpressionEditorModal/ExpressionModalInput.vue

```diff
@@ -33,6 +33,7 @@ export default mixins(workflowHelpers).extend({
 		return {
 			editor: null as EditorView | null,
 			errorsInSuccession: 0,
+			isReadOnly: window !== window.parent && window.parent.location.pathname.includes('/executions'),
```

Comment:
not a great solution.. cannot we simply pass it from parameter input? we already have that info there no?